src/js/Controllers/BCController.js

```diff
@@ -106,10 +106,14 @@ class BCController {
                 if (rpc.params.appCapability.appCapabilityType === 'VIDEO_STREAMING'
                     && rpc.params.appCapability.videoStreamingCapability) {
                     var vsc = rpc.params.appCapability.videoStreamingCapability;
-                    if (!vsc.additionalVideoStreamingCapabilities) {
-                        vsc.additionalVideoStreamingCapabilities = [];
+                    var vsCapabilities = []
```

Comment:
Is there any reason why `vsc` is not included in this list?

Reply:
Ah, seems this format was accepted in https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0332-additional-video-streaming-capabilities-validation.md